notebooks/yolo-detection-classification/yolo-detection-classification.ipynb

```diff
@@ -0,0 +1,670 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    if detect_device.value != "CPU":
</pre><p>Could you try to use intel device for inference for detect and classification models, like it is in yolov11-object-detection (without quantization):</p><p>res = det_model(IMAGE_PATH, device=f"intel:{device.value.lower()}")</p><p> </p><p>It should simplify and reduce the number of extra code lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3080/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>